perl/lib/Sanger/CGP/Vaf.pm

```diff
@@ -2,7 +2,7 @@ package Sanger::CGP::Vaf;
 use strict;
 use Const::Fast qw(const);
 
-our $VERSION = '5.3.2';
+our $VERSION = '5.3.4';
```

Comment:
You need to delete the tag for 5.3.3 from github if the version was not updated.